sources/packages/backend/libs/sims-db/src/entities/student-assessment.model.ts

```diff
@@ -330,6 +330,9 @@ export interface WorkflowData {
     livingWithParents: FormYesNoOptions;
     numberOfParents?: number;
   };
+  dmnValues: {
+    lifetimeMaximumCSLP: number;
```

Comment:
This interface is shared for part-time/full-time but the `lifetimeMaximumCSLP` is part-time specific, hence it should be optional. Does it make sense?